metabase/metabase
# Add a new SDK setting page and it's dependencies

Closes https://github.com/metabase/metabase/issues/47185

### Description
>[!Note]
> Please note that since this PR fixes quite a lot of existing tests, I'd rather add new tests in a separate PR to make reviewing much more easier.

This PR only add UIs for the new embedding SDK settings page. 

### How to verify

With pro license with `embedding` feature flag (we'll use this until we added `embedding-sdk` to the test token)
- Go to embedding settings page, and see that you can configure embedding SDK for react.
- Click `Configure` should bring you to the SDK settings page
- You should be able to modify the input (more logic around disabling would be added once we added `enable-embedding-sdk` in the next task)

With OSS
- Go to embedding settings page, and see that you can try out embedding SDK for react.
- Click `Try it out` should bring you to the SDK settings page
- The input should be disabled.

### Demo

#### EE
- ![image](https://github.com/user-attachments/assets/47936728-74ba-445b-a1b9-fdb6a659087a)
- ![image](https://github.com/user-attachments/assets/7527db0c-dcac-4fa1-9bd8-014d0cc0ff7a)
- ![image](https://github.com/user-attachments/assets/e432ff8f-4a3f-476b-a562-053aa7201ce8)

#### OSS
- ![image](https://github.com/user-attachments/assets/b1e14953-8500-4fe4-9028-b185e1e02020)
- ![image](https://github.com/user-attachments/assets/66db60df-d963-45c4-8bf7-9d162c5b0e52)
- ![image](https://github.com/user-attachments/assets/20dc46ac-fd19-420b-9cb8-b63b19579c01)





### Checklist
